package.json

```diff
@@ -2,25 +2,52 @@
     "name": "png-js",
     "description": "A PNG decoder in JavaScript",
     "version": "1.0.0",
+    "type": "module",
+    "main": "./lib/png-js.cjs",
+    "module": "./lib/png-js.js",
+    "exports": {
+        ".": {
+            "node": {
+                "import": "./lib/png-js.js",
+                "require": "./lib/png-js.cjs"
+            },
+            "default": "./lib/png-js.browser.js"
```

Comment:
Will this need a "browser" path? If I understood node docs correctly `node` takes precedence over `default`, but it also mentions [community conditions definitions](https://nodejs.org/api/packages.html#community-conditions-definitions) which I guess are supported but not part of the standard

Reply:
No. Browser takes the default